### DISCUSSION TITLE
shadcn does not work on mobile?

### DISCUSSION THREAD

**Discussion by SilkePilon** [shadcn-ui/ui#3747] [General]
ive been working on my own app for pc and was checking if my phone would also load it but i found out that the screen goes black if the screen size is below 768 in width???

is this an intended feature. 

(also check on the shadcn website on the examples and they just convert into an image if you resize the screen)

**Comment by CesarQD98** [comment]
Maybe some code can show us what components you have used in your app. Docs examples showcase some situations in which you use some components to build some 'mobile' components. [Here's a example in the Drawer component docs.](https://ui.shadcn.com/docs/components/drawer#:~:text=Examples-,Responsive%20Dialog,-You%20can%20combine)

**Comment by dertuman** [comment]
it seems like it has some issues with iPhone, and some of the issues can also be seen in the demo. I think the library should be thoroughly tested on mobile phones physically, not just with the computer resizing with dev tools. Especially iPhone, is problematic

**Reply by dertuman** [reply]
EDIT: issues for example in the input component (inside a drawer), on iphone, when I click on the input the keyboard goes up and there's a black section on top of it 🗡️ , also the date picker doesn't open (iphone 14 pro on chrome), anyone know how to fix it? screenshot for reference
![image](https://github.com/user-attachments/assets/0cc93bee-6465-471a-8c6c-422d959d4dad)

**Comment by forrestwilkins** [comment]
@dertuman Has this project actually been abandoned? Doesn't appear that way from insights: https://github.com/shadcn-ui/ui/pulse/monthly

**Reply by dertuman** [reply]
I Wil update and check if the iPhone experience is still terrible 🙂

**Comment by ushicode** [comment]
You wanna checkout RNR [RNR](https://rnr-docs.vercel.app/getting-started/introduction/)

That's kinda shadcn for mobile
